android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -978,6 +985,7 @@ private void showSearchBoxTooltip() {
                                 if (viewGroup != null && highlightView != null) {
                                     viewGroup.removeView(highlightView);
                                 }
+                                maybeShowNotificationPermissionRetionale();
```

Comment:
I suppose it's typo here, should be `Rationale`

Reply:
oh yes. good catch.